storage_service/locations/models/package.py

```diff
@@ -1140,19 +1146,22 @@ def create_replica_pointer_file(
             master_compression_event.compression_details
         )
         replica_premis_relationships = [
-            _get_replication_derivation_relationship(
+            premis.get_replication_derivation_relationship(
                 master_aip_uuid, replication_event_uuid
             )
         ]
-        replica_premis_object = replica_package._create_aip_premis_object(
+        replica_premis_object = premis.get_aip_premis_object(
+            replica_package,
             master_checksum_algorithm,
             master_checksum,
             archive_tool,
             compression_program_version,
             premis_relationships=replica_premis_relationships,
         )
-        replica_premis_creation_event = replica_package.get_premis_aip_creation_event(
-            master_aip_uuid=master_aip_uuid, agents=replica_premis_creation_agents
+        replica_premis_creation_event = premis.get_premis_aip_creation_event(
+            replica_package,
```

Comment:
```suggestion
            package=replica_package,
```
Though we're really not very consistent across this module.... :confounded: :nerd_face:

Reply:
I think it makes sense to have a mix of args and kwargs here (args being required kwargs, being optional), but I've removed passing packages at all since they're not really required.